WalletWasabi/Blockchain/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -288,4 +291,30 @@ public void UndoBlock(Height blockHeight)
 	{
 		Coins.SwitchToUnconfirmFromBlock(blockHeight);
 	}
+
+	internal void UpdateRelevantTxs(SmartTransaction tx)
```

Comment:
```suggestion
	public void UpdateRelevantTransactions(SmartTransaction tx)
```
If you go with this, then don't forget to update the line where this is called.